templates/project/widgets/number/number.coffee

```diff
@@ -22,3 +22,8 @@ class Dashing.Number extends Dashing.Widget
         c.replace /\bstatus-\S+/g, ''
       # add new class
       $(@get('node')).addClass "status-#{data.status}"
+
+  parseValue: (number) ->
+    if number == parseInt(number, 10) then parseInt(number) else parseFloat(number)
```

Comment:
If `number` has non-numeric digits, this function will assume it is a float.

For example: `parseInt("15px", 10);` should return 15, but would return the float 15.0 instead.

Not sure if this is a big deal, but it would be confusing if it happened and you weren't aware of this. Seems like an edge case to me though.